atlasdb-cassandra/src/main/java/com/palantir/atlasdb/cassandra/CassandraKeyValueServiceConfig.java

```diff
@@ -47,6 +47,7 @@
 @JsonSerialize(as = ImmutableCassandraKeyValueServiceConfig.class)
 @JsonTypeName(CassandraKeyValueServiceConfig.TYPE)
 @Value.Immutable
+@SuppressWarnings("AutoService")
```

Comment:
Might be worth adding comment that we're suppressing warning that `AutoService` warns because it does not work on interfaces/abstract classes, but these are `@Value.Immutable` and the `@AutoService` will be propagated to the generated classes where auto service will generate the proper metadata for use at runtime.